src/aosm/azext_aosm/definition_folder/reader/bicep_definition.py

```diff
@@ -153,6 +159,72 @@ def _artifact_manifests_exist(
 
         return ManifestsExist.NONE
 
+    @staticmethod
+    def _base_Resources_exist(

```

Comment:
Could you please make the R lowercase to match the style of the rest

Reply:
changed this.